### PR TITLE
Create section

### PR DESCRIPTION
**Pull request by iva585**
created different sections, added function to navbar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,163 +10,287 @@
   </head>
   <body>
     <header>
-      <h1 class="heading">Quiz-App</h1>
-      <h2 class="heading hidden-js">Bookmarks</h2>
-      <h2 class="heading hidden-js">Create</h2>
-      <h2 class="heading hidden-js">Profile</h2>
+      <h1 data-js="home-title" class="heading">Quiz-App</h1>
```
Comment (derjubu) [inline index.html:13]: If you toggle this to hidden-js, there is no h1 on the page anymore. You should make the other headings h2 as well.

File: styles/index.css [hunk 2]
```diff
@@ -127,10 +135,125 @@ ul {
...
+  background-color: #e85a4f;
+  box-shadow: 0 0 17px 5px rgba(233, 128, 116, 0.58);
+  padding: 7px;
+  border-radius: 0.6rem;
+  color: #eae7dc;
+  text-decoration: none;
+  font-size: 0.8rem;
+}
+
+.hidden-js {
```
Comment (derjubu) [inline styles/index.css:257]: I suggest to just name it 'hidden'. I don't think it it is relevant to the naming that you use it in the JS functions.

**Review by derjubu — APPROVED** [review]
Really well done! Just one annotation regarding the h1, that you should look into. Apart from that: 👌

File: styles/index.css [hunk 3]
```diff
@@ -93,6 +88,19 @@ ul {
   height: 100%;
 }
 
+.navbar__icon {
+  fill: #e85a4f;
+}
+
+.navbar__icon--active {
```
Comment (PReichetanz) [inline styles/index.css:95]: The term is very speaking, but according to BEM, it should be "navbar__icon-active" (as you have already done below).

**Review by PReichetanz — COMMENTED** [review]
Nice work!

**Pull request merged**